### PR TITLE
Grids Done

### PR DESCRIPTION
**Pull request by MayurChawla**
I have created Grid with 2 items and 3 items in a grid.

### CODE REVIEW

File: Components/Grids/Grids.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap');
+
+:root {
+    --primary-color : #c2004b;
```
Comment (aurobindoGupta) [inline Components/Grids/Grids.css:4]: use common files instead of setting these colors in every component

**Review by aurobindoGupta — APPROVED** [review]
nice

File: Components/Grids/Grids.css (hunk 1 above)
Comment (MayurChawla) [reply Components/Grids/Grids.css:4]: yes post completing individual components will do that

File: Components/Grids/Grids.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    width: 100%;
+    height: 100%;
+}
+body {
+    background-color: #F7F7F7;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (nareshpoonia) [inline Components/Grids/Grids.css:34]: You can consider using utility class flex-center here

**Review by nareshpoonia — COMMENTED** [review]
Code looks clean. Just one comment on using utility classes./
Good to merge

**Pull request merged**